Makefile

```diff
@@ -688,6 +688,8 @@ CFLAGS += -D_GNU_SOURCE -ggdb3 -Wall -Wfatal-errors -Werror
 
 CPPFLAGS += --std=c++11
 
+LIBS += -ldl
```

Comment:
Seems like this is needed on the Ubuntu Linux (focal) version I'm running on my Docker image.